xla/service/while_loop_unroller.h

```diff
@@ -42,6 +42,15 @@ struct WhileLoopConfig {
   int64_t induction_var_idx;
 };
 
+// Config for unrollable while loops.
+struct UnrollResult {
+  // Whether it's unrolled.
+  bool unrolled = false;
+  // New unrolled while op that replaced the old one.
+  // For non-trivially wrapped case, this would be nullptr.
```

Comment:
This comment is unclear to me. What is the "non-trivially wrapped case"? This is only nullptr if either the while-loop was not unrolled or if `wrap_in_trivial_loop=false`, in which case there is no new while-op, right? If so, phrase something like

> If the while-loop has been unrolled and replaced with a new unrolled while-loop with a trip count of 1, this is the new while-loop. Otherwise this is nullptr.